srcDoc/execution.rst

```diff
@@ -83,6 +83,17 @@ In such case, add the *OpenSSL* libraries in
    ``$DYLD_LIBRARY_PATH``
 -  On Windows systems, add ``%OPENSSLHOME$/<debug or release>/bin`` to
    ``%PATH%``
+   
+When using *RTI Perftest* in *VxWorks* kernel mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After compiling for *VxWorks* (see `Compilation <https://github.com/rticommunity/rtiperftest/blob/master/srcDoc/compilation.rst>`__), load the shared object (.so):
```

Comment:
After compiling for *VxWorks* (...), the shared object need to be loaded in the kernel, by doing: